roles/bootstrap-os/defaults/main.yml

```diff
@@ -1,5 +1,18 @@
 ---
+pip_python_coreos_modules:
```

Comment:
Why are all these modules needed?

Reply:
#1032

Reply:
To my knowledge these are no longer necessary, that issue is from 2017...